unit-tests/openioc_tests.py

```diff
@@ -0,0 +1,155 @@
+# Copyright (c) 2015, The MITRE Corporation. All rights reserved.
+# See LICENSE.txt for complete terms.
+
+import unittest
+import lxml.etree as ET
+
+from StringIO import StringIO
```

Comment:
Packages/modules are usually lowercase (ignore `StringIO` below, heh). I am kinda all over the place in how I import `lxml.etree`, but I think I usually do a `from lxml import etree`.

[Pep8](https://www.python.org/dev/peps/pep-0008/#imports) says the following about import ordering:
```
Imports should be grouped in the following order:

1. standard library imports
2. related third party imports
3. local application/library specific imports

You should put a blank line between each group of imports.
```

So the etree import should follow the `StringIO` import.